css/Components/About.scss

```diff
@@ -0,0 +1,195 @@
+$About-numberCaption-line-height: 26px;
+
+$About-attendancePhoto: "../../images/about/attendance@2x.png";
+$About-venuePhoto: "../../images/about/venue@2x.png";
+
+$About-photo-gradient: 133deg,
+  rgba($Color-deep-cove, 0.4),
+  rgba($Color-madison, 0.4);
+
+$About-photo-alternate-gradient: to right,
+  rgba($Color-deep-cove, 0.4),
+  rgba($Color-deep-cove, 1);
+
+$About-photo-mobile-gradient: to right,
+  rgba($Color-deep-cove, 1),
+  rgba($Color-deep-cove, 0.4);
+
+.About {
+  padding: 0 $Spacing-xxSmall;
+}
+
+.About-header {
+  margin-bottom: $Spacing-base;
+}
+
+.About-content {
+  display: flex;
+  flex-direction: column;
+
+  width: 100%;
+  max-width: $About-content-max-width;
+  margin: 0 auto;
+}
+
+.About-text {
+  max-width: $About-text-max-width;
+  margin: $Spacing-xSmall 0 $Spacing-base;
+
+  &::first-letter {
+    float: left;
+    padding-right: $About-text-first-letter-spacing-right;
+
+    font-size: $About-text-first-letter-font-size;
+    line-height: $About-text-first-letter-line-height;
+  }
+}
+
+.About-info {
+  position: relative;
+
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  @include Breakpoint-mobileOnly {
+    flex-direction: column;
+  }
+}
+
+.About-attendancePhoto {
+  width: 310px;
+  height: 286px;
+
+  margin-right: $Spacing-xSmall;
+  margin-bottom: $Spacing-xSmall;
+
+  background-image: linear-gradient($About-photo-gradient),
+                    url($About-attendancePhoto);
+  background-size: cover;
+
+  @include Breakpoint-mobileOnly {
+    width: 294px;
+    height: 286px;
+
+    margin-top: $Spacing-large;
+    margin-right: 0;
+
+    background-image: linear-gradient($About-photo-mobile-gradient),
+                      url($About-attendancePhoto);
+  }
+}
+
+.About-venuePhoto {
+  z-index: 1;
+
+  width: 310px;
+  height: 390px;
+
+  background-image: linear-gradient($About-photo-alternate-gradient),
+                  url($About-venuePhoto);
+  background-size: cover;
+
+  @include Breakpoint-mobileOnly {
+    width: 294px;
+    height: 403px;
+
+    background-image: linear-gradient($About-photo-mobile-gradient),
+                      url($About-venuePhoto);
+  }
+}
+
+.About-currentEdition {
+  position: absolute;
+  top: 0;
+  right: $Spacing-small;
+  z-index: 1;
+
+  max-width: $About-currentEdition-max-width;
+
+  @include Breakpoint-mobileOnly {
+    right: initial;
+    left: 0;
+  }
+}
+
+.About-currentTitle {
+  @include Breakpoint-mobileOnly {
+    margin-bottom: $Spacing-xSmall;
+  }
+}
+
+.About-numbers {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+
+  font-family: $About-dataNumber-font-family;
+
+  @include Breakpoint-mobileOnly {
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+}
+
+.About-dataNumber {
+  width: 100%;
+  max-width: $About-dataNumber-max-width;
+
+  @include Breakpoint-mobileOnly {
+    margin-bottom: $Spacing-xSmall;
+  }
+}
+
+.About-number {
```

Comment:
Everything related to these number styles: 

![screen shot 2017-03-09 at 17 22 21](https://cloud.githubusercontent.com/assets/4236592/23762132/fe2ee02c-04ec-11e7-8daa-79f5a308a2bb.png)